### PR TITLE
Add multi-format bundle output and update package.json

### LINKED ISSUE #4: Jyutping Improvement
I don't know how you farm those jyutping, 

https://words.hk/faiman/analysis/wordslist.json
https://words.hk/faiman/analysis/charlist.json

but anyway, if you haven't included this method, I think you can try. I'm too lazy to code a new library so I will use your `to-jyutping`.

Just so if you wanna update the dictionary, you can parse all the words from there, for the tokenizer, we can use jieba

https://github.com/hockyy/jieba-cantonese

I've made a script to auto generate jieba user dict to tokenize, so querying jyutping per token can be better, if the result don't exist, fall back to per character jyutping

### PR DESCRIPTION
**Pull request by hockyy**
If want to use the old method of importing

### CODE REVIEW

**Comment by graphemecluster** [comment]
Great, this is just what I am about to do.
Currently a UMD is built, which is usable by CJS but not ESM, so I guess the CJS one is redundant, contrarily?

File: rollup.config.js [hunk 1]
```diff
@@ -25,4 +39,4 @@ export default [
     },
     plugins: [dts(), del({ targets: ["./dist/*", "!./dist/index.*"], hook: "buildEnd" })],
   },
-];
+];
```
Comment (graphemecluster) [inline rollup.config.js]: Please add back the new line 😅

Comment (hockyy) [reply rollup.config.js]: added hhh

**Comment by hockyy** [comment]
> Great, this is just what I am about to do. Currently a UMD is built, which is usable by CJS but not ESM, so I guess the CJS one is redundant, contrarily?

actually im not really sure about the behavior

**Comment by graphemecluster** [comment]
This seems to be the common practice 😉

**Pull request merged**